Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveE.cpp

```diff
@@ -98,55 +128,62 @@ void FiniteDifferenceSolver::MacroscopicEvolveECartesian (
         Real const * const AMREX_RESTRICT coefs_z = m_stencil_coefs_z.dataPtr();
         int const n_coefs_z = m_stencil_coefs_z.size();
 
+        FieldAccessorMacroscopic const Bx_raw(Bx, mu_arr);
+        FieldAccessorMacroscopic const By_raw(By, mu_arr);
+        FieldAccessorMacroscopic const Bz_raw(Bz, mu_arr);
```

Comment:
Since this quantity corresponds to the magnetizing field H, I would suggest a slightly different name:
```suggestion
        FieldAccessorMacroscopic const Hx(Bx, mu_arr);
        FieldAccessorMacroscopic const Hy(By, mu_arr);
        FieldAccessorMacroscopic const Hz(Bz, mu_arr);
```